### ISSUE TITLE
Docker image no work after v1

### ISSUE THREAD

**Issue by leonheart67** [rigon/photo-gallery#16]
Hello, just to point out that docker versions after v1 don't work, I don't have web access, although there doesn't seem to be any errors.

```
2023/06/13 20:08:25 Collections: map[Photos:Photos (/photos)]
2023/06/13 20:08:25 WebDAV will be available at http://localhost:3080/webdav
2023/06/13 20:08:25 Starting server: http://localhost:3080
```

best regards

**Comment by rigon** [comment]
I'm glad you pointed out the issue, it must affect everyone trying out the app for the first time. The issue has to do with the bind address in the docker image.
In v1, I was using the [Gorilla](https://www.gorillatoolkit.org/) web server, but after that I switched to [fiber](https://gofiber.io/). By default Gorilla binds to `0.0.0.0` and fiber to `127.0.0.1`, however the docker port forwarding service needs to be bound to `0.0.0.0`. 

You can change the bind address from the command line with `--host 0.0.0.0`.
I just released a new version with the docker image fixed, please use `v1.3.2` or upgrade `latest`.
Please let me know if that solved the issue.

If you could provide any additional feedback on the project, I would really appreciate 😃 Best regards!

**Comment by leonheart67** [comment]
Ty, all working now, I will come back to you after a few days for give a feedback
